src/animators/easingAnimator.ts

```diff
@@ -72,7 +72,7 @@ module Plottable.Animators {
      * @param {number} startDelay The start delay in milliseconds.
      * @returns {Easing} The calling Easing Animator.
      */
-    public startDelay(startDelay: number): Easing;
+    public startDelay(startDelay: number): this;
     public startDelay(startDelay?: number): any {
```

Comment:
cool, makes sense. looks like a TypeScript flaw if we have to use `any` so much to achieve the nice typings for getter/setter methods like this.